test/test_parser_ntriplesstar.py

```diff
@@ -0,0 +1,96 @@
+
```

Comment:
This file (and other test files which essentially run the test suites) should be in `test/test_w3c_spec/`.